debian/po/pt_BR.po

```diff
@@ -3455,18 +3460,28 @@ msgstr ""
 "Valor com tipo incorreto para o campo \"{key}\":\n"
 "{nested_msg}"
 
-#: ../../uaclient/messages/__init__.py:2544
+#: ../../uaclient/messages/__init__.py:2548
 #, python-brace-format
 msgid "Value provided was not found in {enum_class}'s allowed: value: {values}"
 msgstr ""
 "Valor fornecido não está presente nos valores permitidos de {enum_class}: "
 "{values}"
 
-#: ../../uaclient/messages/__init__.py:2555
+#: ../../uaclient/messages/__init__.py:2559
 #, python-brace-format
 msgid "Error updating ESM services cache: {error}"
 msgstr ""
 
+#: ../../uaclient/messages/__init__.py:2565
+#, python-brace-format
+msgid ""
+"There is a problem with the resource directives provided by {url}\n"
+"These entitlements: {names} are sharing the following directives\n"
+" - APT url: {apt_url}\n"
+" - Suite: {suite}\n"
+"Those fields need to be unique for every entitlement."
```

Comment:
"Those fields" feels a bit vague to me, which fields specifically need to be unique?

Reply:
those are the directives above.
So maybe changing `Those fields` for `These directives` ?

Reply:
My idea was to refer to both `APT url` and `Suite`. Do you think I should rewrite the message to make this more explicit ?

Reply:
"These directives" would resolve it, because then there's no ambiguity about what fields are being referred to.

Since you defined already that APT url and Suite are the directives, I don't think you need to specify both of them again. The original wording, it's not clear if "those fields" refers to "these entitlements" or "the following directives" (that was the confusion I had)